install_latest_blue_onyx.ps1

```diff
@@ -164,59 +182,121 @@ try {
         }
     }
     else {
-        Write-Host "$processName.exe is not running. Proceeding with installation..."
+        Write-Host "$processName.exe is not running. Proceeding with installation..." -ForegroundColor Green
     }
 
     # --- 7. Copy new files into .blue-onyx, overwriting if they exist ---
-    Write-Host "Overwriting existing files in $destinationPath with the new files..."
+    Write-Host "Overwriting existing files in $destinationPath with the new files..." -ForegroundColor Green
     Copy-Item -Path (Join-Path $flattenPath '*') -Destination $destinationPath -Recurse -Force
 
     # Cleanup the temp_unzip folder
     Remove-Item -Path $tempExtractPath -Recurse -Force
 
     # --- 8. Add that folder to the PATH (for the user environment) ---
-    Write-Host "Adding $destinationPath to User PATH..."
-    $userPath = [System.Environment]::GetEnvironmentVariable("PATH", "User")
-    
+    Write-Host "Adding $destinationPath to User PATH..." -ForegroundColor Green
+    $userPath = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+
     if ($userPath -notlike "*$destinationPath*") {
         if ([string]::IsNullOrEmpty($userPath)) {
             $newPath = $destinationPath
         } else {
             $newPath = "$userPath;$destinationPath"
         }
-        
-        [System.Environment]::SetEnvironmentVariable("PATH", $newPath, "User")
+
+        [System.Environment]::SetEnvironmentVariable("PATH", $newPath, "Machine")
```

Comment:
I am torn about this, asking for admin makes this feels more insecure.

Could it be optional, if running with admin then use Machine or if it fails then set user?

Reply:
Testing a popup box option.

Reply:
![image](https://github.com/user-attachments/assets/f4d0da69-5326-4a1c-b9a8-19c8183abda1)

![image](https://github.com/user-attachments/assets/627a6aec-2abd-41f4-8d07-ebed7e52ce2a)

![image](https://github.com/user-attachments/assets/cc789f8a-6528-45ee-9643-00df15a13dfe)

Reply:
When the user selects System wide a new Powershell window pops up RunAs and added the System PATH.